apidoc/Titanium/UI/Tab.yml

```diff
@@ -255,6 +255,18 @@ properties:
     osver: {ios: {min: "7.0"}}
     platforms: [iphone,ipad]
 
+  - name: tintColor
+    summary: Defines the color of the tab icon.
+    type: String
+    since: {iphone: "3.1.3", ipad: "3.1.3", android: "9.1.0"}
+    platforms: [iphone, ipad, android]
+
+  - name: activeTintColor
```

Comment:
This is available on iOS already. I think `tintColor` is the active tint and there is an additional one for the unselected items.

Reply:
```
'TabGroup': {
	activeTabIconTint: 'red',
	tabsTintColor: 'blue',
	unselectedItemTintColor: 'green'
}
```

Reply:
Looks like we have a parity issue then. We should select the property most in keeping with our typical property naming/defaults as the proper one, deprecate the other(s) and ensure parity cross-platform.

The issue here seems to be that the relevant properties do already exist on the TabGroup level, but the new implementation is at the Tab level. I can actually see a use case where you'd prefer to set some tab group default for colors but say have individual tabs use different colors when they're active, so I do think they both have merit.

Reply:
We already have a ticket written for `activeTabIconTint` here...
https://jira.appcelerator.org/browse/TIMOB-26734

If I remember right, Android didn't have a native API that supported this for top tabs at the time. It was only possible via style/theme XML settings. But there may be new APIs we can leverage now that we've migrated to AndroidX.

Reply:
It's kind of a mess:

###### TabGroup
- `activeTabIconTint`
- `tabsTintColor` (apparently applies to title and icons)
- `tintColor`
- `unselectedItemTintColor` (doesn't specify if this applies to title and/or icons)

###### Tab
- `tintColor`
- `activeTitleColor`
- `titleColor`

I added `tintColor` and `activeTintColor` to `Tab` as this is where the tab title colors are also defined. But there are a number of properties that can cause confusion and conflict. There's also a number of iOS specific properties that are exposed that could have cross-platform support with Android.

Reply:
I'm going to mark this as a WIP and will clean up parity with iOS.

Reply:
Thanks Gary! Thats really worth it and a big improvement. And sorry for noising here 😄